graspy/inference/latent_position_test.py

```diff
@@ -78,11 +78,11 @@ class LatentPositionTest(BaseInference):
         The observed difference between the embedded positions of the two input graphs
         after an alignment (the type of alignment depends on ``test_case``)
 
-    p_value_1_, p_value_2_ : float
+    p_value_1\_, p_value_2\_ : float
         The p value estimated from the null distributions from sample 1 and sample 2.
 
     p_value_ : float
```

Comment:
can you explain why this one wouldn't need an escape char, but the others did? what was the warning being thrown here?

Reply:
Honestly, I'm not sure...Something buggy with Sphinx I think.  The exact errors were:
```11:02:00 AM: /opt/build/repo/graspy/inference/latent_position_test.py:docstring of graspy.inference.LatentPositionTest:64: WARNING: Unknown target name: "p_value_1".

11:02:00 AM: /opt/build/repo/graspy/inference/latent_position_test.py:docstring of graspy.inference.LatentPositionTest:64: WARNING: Unknown target name: "p_value_2".```

Reply:
and what you did fixed it? or was there somewhere in the code where `p_value_1` was listed instead of `p_value_1_` or something like that?

Reply:
Yes, what I did fixed it, at least locally.  I didn't alter any code, just added the \'s in the docs.